assembl/models/preferences.py

```diff
@@ -807,12 +809,14 @@ def get_preference_data_list(self):
                     P_OVERRIDE_SOCIAL_AUTOLOGIN,
                     P_SEND_SYNTHESIS,
                     P_VOTE,
+                    P_ADD_SIDE_COMMENT,
                 ],
                 R_PARTICIPANT: [
                     P_ADD_POST,
                     P_DELETE_MY_POST,
                     P_EDIT_MY_POST,
                     P_VOTE,
+                    P_ADD_SIDE_COMMENT,
```

Comment:
Cool, since the base Participant will have this perm, I suggest the the `R_CATCHER` to also have this perm as well.